client/src/components/Navbar.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useContext, useState } from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/core/styles";
+import { AppBar, Toolbar, Button, Menu, MenuItem, Avatar, Badge } from "@material-ui/core";
+import NotificationsNoneIcon from "@material-ui/icons/NotificationsNone";
+import UserContext from "../context/UserContext";
```

Comment:
I think I mentioned this in another PR (from Gerson), lets avoid relative paths that go backwards)

Reply:
Fixed